\chapter{Tail-Anchored Proteins Revisited; An Up-To-Date Dataset And Biochemical Insights Into Spontaneous Insertion} %Mutants outlined + any results from Abbi
\section{Abstract}

\section{Introduction}
This study aims to identify \gls{snare} proteins in eukaryotic proteomes by filtering through large datasets using automatically predicted TrEMBL consensus, and manually annotated SWISS-PROT transmembrane regions. The pipeline generates a list of singlepass proteins with a transmembrane domain close to the C terminal, that are not splice isoforms. A previous study predicted 411 tail anchor proteins~\cite{Kalbfleisch2007}.

\section{Methods}
The original list UniProt protein database was queried for records containing ``TRANSMEM'' annotation on June 15, 2016, totaling 75826 records from swissprot, and 12322000 records from TrEMBL.

%An executable version of the program TMPRED (for MacOS X) was provided by Dr Laurent Falquet of the Swiss Institute of Bioinformatics. A FASTA sequence for each protein in our original list [NCBI nonredundant protein database (ftp://ftp.ncbi.nih.gov/genomes/H_sapiens/protein/protein.fa.gz) queried on June 21, 2007, totaling 34 180 proteins] was run through the TMPRED software. Those predicted transmembrane regions with a score greater than or equal to our threshold of 1500 were identified. From these results, those proteins were collected that had a single predicted trans- membrane segment. This list comprised 5028 proteins. This list was further filtered by identifying those proteins whose predicted transmembrane segment terminated within the final 25 amino acids of the protein. The resulting list had 644 proteins remaining. A FASTA file was created containing the first 70 amino acids for each of these 644 proteins. This file was run through the program SIGNALP (13) to identify those proteins that were predicted to contain a signal sequence for import through the ER translocon. The hidden Markov model prediction was used, and an Sprob threshold of 0.750 was used as a maximum score. This reduced the list to 429 proteins. FASTA sequences were created that contained the first 70 amino acids of these proteins. They were fed into the MITOPROT (15) program where a probability was calculated for the import of each of these proteins into mitochondria. A maximum probability threshold of 0.850 was estab- lished, and 411 proteins remained in the list after filtering. The proteins XP_001125749.1, XP_001133750.1 and XP_001134217.1 each began with an ambiguous X. For the purposes of this search, this was changed to a methionine in order to satisfy the requirements of the MITOPROT program. Thresholds were determined as outlined in Results and Discussion.
%Total hydrophobicity of each TMPRED-predicted transmembrane segment was calculated by summing the hydrophobicity values, as calculated by Kyte and Doolittle (28).
%Twenty dimensional vectors were used to represent the amino acid composition of the transmembrane segment of each protein. For each di- mension, the normalized frequency was calculated by counting the number of occurrences of each amino acid within a transmembrane segment (as defined by TMPRED) and dividing each count by the total number of amino acids in the transmembrane segment. The protein ambiguity code ‘X’ was found within some of the transmembrane segments. Its contribution was calculated andwas given the same weight as any of the unambiguous amino acids. For each class of protein (Figure 2 and Tables S2 and S3), a represen- tative average vector was calculated by summing the contributions for each amino acid. The average vector was then reduced to unit length by dividing by the square root of the sumof the squares of all theamino acidscumulative values. Prior to calculation of the dot product, each protein’s amino acid contribution vectorwas reduced to unit length by dividing by the square root of the sum of the squares of each amino acid contribution. The dot products were then calculated by multiplying each individual amino acid contribution of a specified representative average against each analogous amino acid contribution of a specific protein’s transmembrane region. These values were then summed to produce the representative value reflecting the coincidence of amino acid distributions.
Expression
\subsection{Filtering the Uniprot database}
Steps carried out by Kalbfleisch \textit{et al.} published in Traffic 2007 (8: 1687\-1694)~\cite{Kalbfleisch2007}, were recreated using up to date tools. The non\-redundant human dataset of 145,715 proteins from SwissProt and TrEMBL~\cite{TheUniProtConsortium2014}. 2,478 singlepass proteins were programmatically extracted according to the TRANSMEM count from that list. Then TMDs not within 15AA of the C terminal were removed, resulting in 455 proteins. No splice isoforms were detected according to searching for NON\_TER annotation. 195 proteins of the 411 predicted proteins from the previous study were successfully mapped using the Uniprot mapping tools~\cite{TheUniProtConsortium2014}. Duplicate IDs from the previously predicted tail anchored protein were removed from the set. The remaining dataset contained XXX proteins.

\subsection{Calculating Hydrophobicity}

\subsection{Calculating Sequence Complexity}


\section{Results}

\subsection{An Up To Date Tail-Anchor Dataset}

\subsection{Potential Tail-Anchored SNARE Protein Discovery}

\subsection{Biology of Spontaneously Inserting Tail Anchored Proteins}
